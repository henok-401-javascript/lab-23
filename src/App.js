import React from 'react';
import Header from './components/header';



class App extends React.Component{
  render(){
    return (
      <div className="App">
       
         <Header/>
    
      </div>
    );


  }
}

export default App;
