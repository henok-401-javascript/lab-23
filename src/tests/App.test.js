import React from 'react';
import App from '../App.js';
import renderer from 'react-test-renderer';


describe('snapShot test' , () => {
  it('It can Successfully check by using snapshot' , () =>{
   let htmlSnap = renderer.create(<App/>).toJSON();

   expect(htmlSnap).toMatchSnapshot();
  })
})