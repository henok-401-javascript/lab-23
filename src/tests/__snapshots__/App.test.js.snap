// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapShot test It can Successfully check by using snapshot 1`] = `
<div
  className="App"
>
  <div
    className="resty-body"
  >
    <div
      className="form-input"
    >
      <header>
        <h1>
          RESTy
        </h1>
      </header>
      <input
        className="input-field"
        onChange={[Function]}
        type="text"
        value=""
      />
      <div>
        <select
          className="lableButton"
          onChange={[Function]}
        >
          <option
            value="get"
          >
            GET
          </option>
          <option
            value="post"
          >
            POST
          </option>
          <option
            value="put"
          >
            PUT
          </option>
          <option
            value="delete"
          >
            DELETE
          </option>
        </select>
        <button
          className="button"
          onClick={[Function]}
        >
          Submit
        </button>
      </div>
    </div>
    <div
      className="responseBody"
    >
      <div
        className="header"
      >
        <pre
          className="headerClass"
        >
          {}
        </pre>
        <div
          className="body"
        >
          <pre
            className="bodyClass"
          >
            {}
          </pre>
        </div>
      </div>
    </div>
  </div>
</div>
`;
